src/layout.jl

```diff
@@ -23,7 +23,7 @@ function flex(elem=nothing)
     style(elem, "display"=>"flex")
 end
 
-function container(xs...)
+function container(xs::AbstractVector)
```

Comment:
This was necessary because after the `Node` -> `node` change, this would otherwise create a `PersistentVector{Vector{Node}}` instead of a `PersistentVector{Node}`.

Reply:
Ah, this fixes https://github.com/JuliaGizmos/CSSUtil.jl/issues/5.

Reply:
Nice! Probably worth adding a `container(xs...) = container([xs...])` method as well: this function is exported so we should try to preserve the multi-argument method.